web/schemas/ID4B.json

```diff
@@ -395,16 +377,16 @@
     },
     {
         "key": "Calibration",
-        "type": "bool",
+        "type": "list_str",
         "optional": false,
         "multiple": false,
         "section": "Sample",
         "description": "Specify your calibration sample (e.g. CeO2, LaB6)?",
         "placeholder": "CeO2",
         "value": [
-            "",
-            "true",
-            "false"
+            "CeO2",
```

Comment:
Since you removed empty value, it will use a first one as default, i.e. the default will be Ce02, is it what you want? If not, put back empty value as first element of the list and then in drop-down menu nothing will be selected and user may need to make a choice.

Reply:
yes, CeO2 is the standard ..